src/components/table/examples/table.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component, Host, h, Prop, Element } from '@stencil/core';
+import { Column } from '../table';
+
+@Component({
+    tag: 'limel-example-table',
+    shadow: true,
+})
+export class TableExample {
+    public tableData: any[] = [
+        {
+            id: 1,
+            name: 'Billy Bob',
+            age: 12,
+            gender: 'male',
+            height: 95,
+            col: 'red',
+            dob: '14/05/2010',
```

Comment:
I think we should avoid abbreviations like this, even if this is just an example component